modules/serverless-ossm-enable-sidecar-injection-with-kourier.adoc

```diff
@@ -1,43 +1,38 @@
-// Module included in the following assemblies:
-// * serverless/networking/serverless-ossm.adoc
-// * serverless/networking/serverless-ossm-jwt.adoc
-
-[id="serverless-enable-sidecar_{context}"]
+[id="serverless-ossm-enable-sidecar-injection-with-kourier_{context}"]
 = Enabling sidecar injection for a Knative service
 
 You can add the `sidecar.istio.io/inject="true"` annotation to a Knative service to enable sidecar injection for that service.
 
 [IMPORTANT]
 ====
-Adding sidecar injection to pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
+Adding sidecar injection to pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported when Kourier is enabled.
```

Comment:
Do you need commas here?
_system namespaces, such as `knative-serving` and `knative-serving-ingress`,_